components/Footer.js

```diff
@@ -7,6 +7,10 @@ export default function Footer() {
 					<a href="https://aurooba.com" target="_blank" rel="noopener">
 						Aurooba
 					</a>
+					{" "}and{" "}
```

Comment:
This feels a bit disjointed, could we improve this so the spaces aren't separate from the `and`?